tests/st/libnetwork/test_labeling.py

```diff
@@ -81,9 +81,7 @@ def setUpClass(cls):
         cls.hosts.append(cls.host2)
 
         for host in cls.hosts:
-            start_calico_node(host,
-                              ["CALICO_LIBNETWORK_LABEL_ENDPOINTS=true",
-                               "CALICO_LIBNETWORK_CREATE_PROFILES=false"])
+            host.start_calico_node(options='--use-docker-networking-container-labels')
```

Comment:
I wanted to make sure I addressed the correct thing.  Are you referring to removing the method definition at the bottom of the file?  I believe the place you commented still requires the `host.start_calico_node` call, if I have misunderstood please let me know.  Thanks.